modules/machineset-yaml-gcp.adoc

```diff
@@ -58,15 +89,25 @@ spec:
             creationTimestamp: null
           networkInterfaces:
           - network: <infrastructureID>-network <1>
-            subnetwork: <infrastructureID>-<role>-subnet <2>
+            subnetwork: <infrastructureID>-<role>-subnet <1> <2>
```

Comment:
should be `subnetwork: <infrastructureID>-worker-subnet`

Reply:
@sunzhaohua2 Is this true for machine sets on Infra and GCP?

Reply:
@mburke5678 I think so, as there is no subnet `<infrastructureID>-infra-subnet ` in cluster,  if we set it like this, the machine will fail to create.